bigtable-hbase-parent/bigtable-hbase/pom.xml

```diff
@@ -38,28 +49,10 @@ limitations under the License.
             <version>${project.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>com.google.protobuf</groupId>
-            <artifactId>protobuf-java</artifactId>
-            <version>${protobuff-java.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>io.grpc</groupId>
-            <artifactId>grpc-core</artifactId>
-            <version>${grpc.version}</version>
-        </dependency>
-
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>${guava.version}</version>
-        </dependency>
```

Comment:
I don't understand why these are being removed

Reply:
It was probably a mistake.  I'll reintroduce these, and just remove the versions.